### ISSUE TITLE
Add some form of backward stepping to Truffle Debugger

### ISSUE THREAD

**Issue by haltman-at** [trufflesuite/truffle#3854]
As stated.  This would need a lot of figuring out.

**Comment by mnaei** [comment]
I was hoping to find something like the inverse of "n" command. 

"y" and "Y" are still useful.

**Comment by haltman-at** [comment]
Yes, to be clear, the intent of the issue is that there should be an inverse of `n`; that's why it's still open.
